ee_tests/src/page_objects/launcher.page.ts

```diff
@@ -32,6 +35,7 @@ export class CreateApplicationPage {
 
     async clickContinue(): Promise<void> {
         let continueButton = new Button(element(by.id('cancelImportsButton')), 'Continue');
+        await browser.sleep(5000);
```

Comment:
Do not use browser.sleep unless there is no other workaround. In that case, use some github issue to describe why it is necessary, for example like here:

https://github.com/fabric8io/fabric8-test/blob/58f8ed31058ed1f9e2731640bdade0ef87c418c8/ee_tests/src/page_objects/launcher.page.ts#L21

Reply:
This browser.sleep has been used here as no other method was working and for the similar purpose as mentioned in the above link

Reply:
That is OK just add the link to GitHub so that we can keep track of all browser.sleeps.